src/main/java/com/clxcommunications/xms/api/RecipientDeliveryReport.java

```diff
@@ -116,11 +116,12 @@
 	public abstract OffsetDateTime at();
 
 	/**
-	 * The message timestamp as recorded by the network operator.
+	 * The message timestamp as recorded by the network operator, if message dispatched.
 	 * 
-	 * @return a date and time
+	 * @return a date and time if message dispatched; <code>null</code> otherwise
```

Comment:
`<code>null</code>` can be replaced with `{@code null}`, which is shorter.

Reply:
Yes. But wanted to have same format through out the project. May be will update the entire project in future improvements.